dali/base/dadfs.cpp

```diff
@@ -9567,7 +9571,7 @@ class CDaliDFSServer: public Thread, public CTransactionLogTracker, implements I
         StringArray authScopes;
         CIArrayOf<CFileMatch> matchingFiles;
         start = msTick();
-        count = scanner.getResults(auth, udesc, matchingFiles, authScopes);
+        count = scanner.getResults(auth, udesc, matchingFiles, authScopes, false);
```

Comment:
What was the exact error?
Did it run out of physical memory or did it exceed this fixed sized buffer (100MB) ?